src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.ps1

```diff
@@ -177,6 +181,132 @@ function Test-AnalysisServicesServerScaleUpDown
 	}
 }
 
+<#
+.SYNOPSIS
+Tests firewall feature of Analysis Services server
+#>
+function Test-AnalysisServicesServerFirewall
+{
+	try
+	{  
+		# Creating server
+		$location = Get-Location Microsoft.AnalysisServices
```

Comment:
You need to supply all three parameters: Get-Location -ProviderNamespace -ResourceType -preferredLocation

Reply:
done